index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <!-- <h1>hello word!, we are software developers!</h1> -->
+      <h1 id="title">Survey Form</h1>
+      <p id="description">This is a survey form to submmit to the bootcamp</p>
```

Comment:
This section could be included in a header tag for better organization

Reply:
- [ ] please i kindly suggest you try to sound polite words when requesting for changes from the student, you can use words such as `i kindly suggest, i think you can, please ensure you` :+1: :sparkles: